stable/express-gateway/values.yaml

```diff
@@ -3,7 +3,7 @@
 
 image:
   repository: expressgateway/express-gateway
-  tag: v1.11.0
+  tag: v1.12.1
```

Comment:
can you please update the readme to match? TIA

Reply:
@davidkarlsen Done!